vector/src/main/java/im/vector/app/features/spaces/invite/SpaceInviteBottomSheetViewModel.kt

```diff
@@ -57,6 +61,38 @@ class SpaceInviteBottomSheetViewModel @AssistedInject constructor(
                         peopleYouKnow = Success(peopleYouKnow)
                 )
             }
+            refreshInviteSummaryIfNeeded(roomSummary)
+        }
+    }
+
+    private fun refreshInviteSummaryIfNeeded(roomSummary: RoomSummary) {
+        if (roomSummary.membership == Membership.INVITE) {
```

Comment:
To follow @ganfra 's remark, I would have let this `if` test at the calling place (i.e. in the init {} block) to reduce indentation in the body of this method. Also the method could then be renamed to `fetchRoomSummary`